docs/_pages/releases.md

```diff
@@ -15,6 +15,7 @@ sidebar:
 * Added `ObjectAssertions<TSubject, TAssertions>` to ease creation of custom assertion classes - [#1371](https://github.com/fluentassertions/fluentassertions/pull/1371).
 * Added `ComparingBy{Members,Value}(Type)` to allow specifying open generic types - [#1389](https://github.com/fluentassertions/fluentassertions/pull/1389).
 * Added overload of `CollectionAssertions.NotBeEquivalentTo` that takes a `config` parameter` - [#1408](https://github.com/fluentassertions/fluentassertions/pull/1408).
+* Made StringAssertions.StartWith and String.StartsWith consistent - [#1413](https://github.com/fluentassertions/fluentassertions/pull/1413).
```

Comment:
Would be nice to rephrase that in such a way that it's clear that this is about empty strings.

Reply:
@dennisdoomen maybe something like this?
Changed `StringAssertions.StartWith`, `StringAssertions.EndWith` and their `EquivalentOf` versions to allow empty strings
Could you advise?

Reply:
Changed to that string for now. Feel free to propose another one, I'll commit your one then.